_posts/2022-03-10-jump-1.0.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: post
+title:  "JuMP 1.0.0 is released"
+date:   2022-03-10
+categories: [releases]
+permalink: /blog/1.0.0-release/
+---
+
+_Authors: Miles Lubin, Oscar Dowson, Iain Dunning, Joaquim Dias Garcia, Joey Huchette, and Benoît Legat._
+
+We are happy to announce the release of JuMP 1.0!
+
+Nearly 10 years in the making, the release of JuMP 1.0 represents a major
+milestone in the history of JuMP. It also represents a stable platform from
+which we can continue to build new and exciting features for a long time into
+the future.
+
+In this post we explain what "going 1.0" means for us now and what it will mean
+for the future.
+
+## What is JuMP?
+
+JuMP is a modeling language and collection of supporting packages for
+[mathematical optimization](https://en.wikipedia.org/wiki/Mathematical_optimization)
+problems in [Julia](https://julialang.org).
+
+JuMP makes it easy to formulate and solve a range of problem classes, including
+linear programs, integer programs, conic programs, semidefinite programs, and
+constrained nonlinear programs.
+
+JuMP is one of the [10 most popular Julia packages](https://juliahub.com/ui/Packages),
+and it has tens of thousands of users in countries worldwide, with over 50
+thousand downloads in the last six months.
+
+JuMP consists of two main Julia packages, [JuMP.jl](https://github.com/jump-dev/JuMP.jl)
+and [MathOptInterface.jl](https://github.com/jump-dev/MathOptInterface.jl),
+along with a plethora of supporting packages which interface MathOptInterface.jl
+to external solvers, use JuMP to solve application specific optimization problems,
+or extend JuMP to new problem domains.
+
+JuMP-dev, the annual user conference, has dozens of contributed talks, and there
+are over 250 Julia packages that depend on JuMP and MathOptInterface for things
+such as
+[power systems modeling](https://github.com/lanl-ansi/PowerModels.jl),
+[exa-scale metabolic modeling](https://github.com/LCSB-BioCore/COBREXA.jl),
+[convex programming](https://github.com/jump-dev/Convex.jl) and much more.
+
+## A brief history of JuMP
+
+* 09 Oct 2012 - First [commit](https://github.com/jump-dev/JuMP.jl/commit/688b732c13abf7182e438ca22b0b00890a08be63) to JuMP.jl repo
+* 24 Oct 2013 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.1.1) - First public release
+* 15 Dec 2013 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.2.0) - First support for solver callbacks
```

Comment:
There's nothing particularly useful at https://github.com/jump-dev/JuMP.jl/releases/tag/v0.2.0. Link to the release notes instead?